src/main/resources/README-CONTACT-ME.md

```diff
@@ -0,0 +1,124 @@
+
+   * https://prod-15.westcentralus.logic.azure.com:443/workflows/9024092aefba434c9db98dc2536423f6/triggers/manual/paths/invoke?api-version=2016-06-01&sp=%2Ftriggers%2Fmanual%2Frun&sv=1.0&sig=SggEKOTUpII26Oc6XYVl7nbSgluiYdVeIXkq19LuTtY
+   
+
+### Microsoft 365 integration
+
+* No, my SaaS offer does not integrate with Microsoft Graph
+
+* No, I do not have published Teams apps, Office add-ins, or SharePoint Framework solutions that I would like to link to this SaaS offer
+
+# Properties
+
+## Categories
+
+* Primary category: Compute
+
+   * Subcategory: Application Infrastructure
+
+## Industries
+
+* Checked Apps 
+
+* Not checked Microsoft Clouds for Industry
+
+* App version: blank
+
+## Legal
+
+* Checked Use the Standarard Contract for Microsoft's commercial marketplace?
+
+## Custom amendments...
+
+No content.
+
+# Offer Listing
+
+## Name
+
+IBM WebSphere Product Family on Azure Overview
+
+## Search results summary
+
+The IBM WebSphere product family is a suite of enterprise Java application servers.
+
+## Description (Observed max 2973 characters, not including formatting)
+
+<p>The WebSphere product family powers some of the most mission critical enterprise applications across the globe and includes WebSphere (Traditional) Application Server, WebSphere Liberty and Open Liberty.</p>
+
+<p>WebSphere products are key components in enabling enterprise Java workloads on Azure. IBM and Microsoft are collaborating on a set of jointly developed and supported solutions for the product family. The offers cover a range of use cases from mission critical existing traditional workloads to cloud-native applications. Offers include Open Liberty on Azure Red Hat OpenShift (ARO), WebSphere Liberty on ARO, Open Liberty on the Azure Kubernetes Service (AKS), WebSphere Liberty on AKS, and WebSphere Application Server (Traditional) on Virtual Machines. The offers make it as easy as possible to migrate your application to the cloud by automating most boilerplate Azure and Java resource provisioning and configuration tasks. Once initial provisioning is complete, you are completely free to customize deployments further.</p>
+
+<p>The currently available offers are linked in the learn more section at the bottom of this page.</p>
+
+<p>IBM and Microsoft also provide basic step-by-step guidance on getting started with <a href=”https://techcommunity.microsoft.com/t5/apps-on-azure/run-websphere-liberty-and-open-liberty-on-openshift/ba-p/1951071”>Liberty and ARO</a>, <a href=”https://techcommunity.microsoft.com/t5/apps-on-azure/run-websphere-liberty-and-open-liberty-on-aks/ba-p/2156486”>Liberty and AKS</a> and <a href="https://community.ibm.com/community/user/wasdevops/blogs/ryan-storey/2021/10/12/twas-azure">WebSphere Traditional on Azure VMs</a>. The guidance is suitable for customers that wish to remain as close as possible to a native OpenShift or Kubernetes manual deployment experience, without relying on an Azure Marketplace offer.</p>
```

Comment:
I'm still struggling a bit with what this is trying to say `The guidance is suitable for customers that wish to remain as close as possible to a native OpenShift or Kubernetes manual deployment experience, without relying on an Azure Marketplace offer.`.  Also, unfortunately, now we have the tWAS on VMs link, it's seems a little out of place.